.gitignore

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Thank you for setting up your gitignore properly!  Makes my job so much easier  :)